docs/users/overview.adoc

```diff
@@ -135,3 +135,12 @@ The example of Process Data for different tasks can be as follows.
 ** RPA robot data
 ** external API call
 ** ...
+
+=== Overlays
+
+`bpmn-visualization` provides https://process-analytics.github.io/bpmn-visualization-js/api/classes/bpmnelementsregistry.html#addoverlays[API to add overlays] on top of BPMN elements. This additional overlays layer permits you to show whatever data you want and enrich your diagrams.
```

Comment:
⚠️ we could use a relative link as we do above in the page. This allow to test changes locally.